### PR TITLE
Fixing the migrations.

### PR DESCRIPTION
**Pull request by burzum**
http://phinx.readthedocs.org/en/latest/migrations.html#the-change-method

> When creating or updating tables inside a change() method you must use the Table create() and update() methods. Phinx cannot automatically determine whether a save() call is creating a new table or modifying an existing one.

### CODE REVIEW

File: config/Migrations/20150412195500_create_tags_tags.php [hunk 1]
```diff
@@ -25,7 +25,6 @@ public function change()
             'default' => 0,
             'length' => 11,
             'null' => false,
-            'unsigned' => true,
```
Comment (jadb) [inline config/Migrations/20150412195500_create_tags_tags.php]: Hmm, not working on mysql?

File: config/Migrations/20150412195500_create_tags_tags.php [hunk 2]
```diff
@@ -35,5 +34,6 @@ public function change()
             'default' => null,
             'null' => true,
         ]);
+        $table->create();
```
Comment (jadb) [inline config/Migrations/20150412195500_create_tags_tags.php]: Derp me.

File: config/Migrations/20150412195501_create_tags_tagged.php [hunk 3]
```diff
@@ -29,5 +29,6 @@ public function change()
             'default' => null,
             'null' => true,
         ]);
+        $table->create();
     }
-}
\ No newline at end of file
+}
```
Comment (jadb) [inline config/Migrations/20150412195501_create_tags_tagged.php]: New line...

File: config/Migrations/20150412195500_create_tags_tags.php (hunk 1 above)
Comment (burzum) [inline config/Migrations/20150412195500_create_tags_tags.php]: Not working in Phinx, the more I use it the more I dislike that lib. :(

* https://github.com/robmorgan/phinx/issues/48
* https://github.com/robmorgan/phinx/pull/40

File: config/Migrations/20150412195501_create_tags_tagged.php (hunk 3 above)
Comment (burzum) [inline config/Migrations/20150412195501_create_tags_tagged.php]: This is not even detected by phpcs? phpstorm and the diff-viewer show a new line for me at the end? I've checked https://travis-ci.org/UseMuffin/Tags/jobs/59320882 and fixed some of the issues it has in the last commit but there is some code that looks like it was intentionally written the way it is but causes phpcs to cry.

File: config/Migrations/20150412195500_create_tags_tags.php (hunk 1 above)
Comment (ADmad) [inline config/Migrations/20150412195500_create_tags_tags.php]: Based on the [docs](http://docs.phinx.org/en/latest/migrations.html) the options key is `signed`, so probably need to use `'signed' => false`.

File: config/Migrations/20150412195501_create_tags_tagged.php (hunk 3 above)
Comment (ADmad) [inline config/Migrations/20150412195501_create_tags_tagged.php]: @jadb New line was missing earlier and the PR adds it, which is fine.

File: config/Migrations/20150412195501_create_tags_tagged.php [hunk 4]
```diff
@@ -11,12 +11,12 @@ public function change()
...
             'null' => true,
         ]);
-        $table->addColumn('entity_id', 'integer', [
+        $table->addColumn('fk_id', 'integer', [
             'default' => null,
             'length' => 11,
             'null' => true,
         ]);
-        $table->addColumn('entity', 'string', [
+        $table->addColumn('fk_model', 'string', [
```
Comment (ADmad) [inline config/Migrations/20150412195501_create_tags_tagged.php]: The [code](https://github.com/UseMuffin/Tags/blob/master/src/Model/Behavior/TagBehavior.php#L116) expects field `fk_table`.

**Comment by ADmad** [comment]
Closing as I think PR #7 takes care of all issues this PR meant to fix. Please open a new PR if something still needs to be taken care of.

**Pull request closed without merging**